server-php/model.crud.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Data model Php web service of DAMAS software (damas-software.org)
+ *
+ * output = JSON
+ * errors = HTTP
+ *
+ * Author Remy Lalanne
+ * Copyright 2005-2014 Remy Lalanne
+ *
+ * This file is part of damas-core.
+ *
+ * damas-core is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * damas-core is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with damas-core.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+session_start();
+
+include_once "lib/http_service.php";
+include_once "lib/model_json.php";
+
+damas_service::init_http();
+damas_service::accessGranted();
+
+header('Content-type: application/json');
+
+$ret = false;
+
+//CRUD Operations
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST')
+{
+  damas_service::allowed( "model::create" );
+  if(is_null($_POST))
```

Comment:
I think $_POST can be empty but cannot be null here. Please verify also for $_GET